vendor/wagn

```diff
@@ -1 +1 @@
-Subproject commit bee8b896f05495d9eb741b048bedc663c911ba3e
+Subproject commit 03cb9f7cc83831c3c0b69e5032fcbe17067a7e28
```

Comment:
I think we probably actually want to bring this back to the commit that was in the original PR.  I updated the test on semaphore NOT to use the latest wagn code but respect the commit indicted in the submodule.

The latest wagn code is a bit ahead of wikirate master now.